charts/system/kubelet-csr-approver/values.yaml

```
@@ -1,18 +1,116 @@
 image:
-  repository: tccr.io/tccr/alpine
+  repository: ghcr.io/postfinance/kubelet-csr-approver
   pullPolicy: IfNotPresent
-  tag: latest@sha256:ade0065e19edaa4f6903d464ee70605111a48394536deb94f31b661264704558
+  tag: 1.2.2@sha256:fdccaa3f2e4f59001b99357565bc5995393c53b21074da769fa53620b5138b85
 
 service:
   main:
-    enabled: false
     ports:
       main:
-        enabled: false
+        port: 8080
+        targetPort: 8080
+        protocol: tcp
 
 workload:
   main:
-    enabled: false
+    enabled: true
+    spec:
+      containers:
+        main:
+          args:
+            - -metrics-bind-address
+            - ":{{ .Values.metrics.port }}"
+            - -health-probe-bind-address
+            - ":8081"
+            - -leader-election
+            - -level
+            - -0
+          probes:
+            liveness:
+              path: "/healthz"
```

```suggestion
            liveness:
              path: "/healthz"
            readiness:
              path: "/healthz"
            startup:
              path: "/healthz"
```

Please configure all paths!